example/android/SNetDemo/app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        android:networkSecurityConfig="@xml/network_security_config">
+        android:networkSecurityConfig="@xml/network_security_config"
+        android:requestLegacyExternalStorage="true">
```

Comment:
Previously we agreed that it is not required? Is it added intentionally?

Reply:
Yes, last time the problem was at initialization, but I didn't test camera capture on Android10. It's not working without that request. We can use it as a short-term fix.
Here's a similar problem described
https://stackoverflow.com/a/58379655